docs/pages/billing/usage-based-pricing.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Usage-based pricing
+description: Learn how Expo applies usage-based billing for customers who exceed their plan limits and how to monitor your EAS Build usage.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CreditCard02Icon } from '@expo/styleguide-icons';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+Expo applies usage-based billing for customers who exceed their plan allowances. This enables our customers to use what they need without worrying about limitations or requiring contractual obligations.
+
+Usage-based billing is enabled for EAS Build and EAS Update and is billed monthly. We provide an estimate of your existing usage and any overage charges on your account's [Billing](https://expo.dev/settings/billing).
+
+## How usage-based pricing works
+
+### EAS Build
+
+For EAS Build, a flat fee is charged for an individual build executed at higher-priority levels. This is totaled monthly and charged at the end of your billing period or sooner if you cancel your plan.
+
+> **Note**: Builds that are canceled before any work is done are not charged.
+
+[Production, Enterprise, and Legacy plans](/billing/plans/#plans) subscribers receive credits for EAS Build. These credits can be used to offset the cost of builds. They are reset at the start of the billing cycle and expire at the end of that billing cycle.
+
+For example, consider an account subscribed to the Production plan which does 15 medium Android builds, and 10 large iOS builds in a billing cycle:
+
+| Description             | Price | Quantity | Total |
+| ----------------------- | ----- | -------- | ----- |
+| Android builds (medium) | $1    | 15       | $15   |
+| iOS builds (large)      | $4    | 10       | $40   |
+| EAS Build Credit        |       |          | -$55  |
+| **Total**               |       |          | $0    |
```

Comment:
Is there a way to right-align all the numbers?

Reply:
Yes we can (by adding `:` to the hyphens of each column, in this case for numbers).

Here is the result after adding `:` (colon):

![CleanShot 2024-03-26 at 02 37 46@2x](https://github.com/expo/expo/assets/10234615/07639398-ee64-400f-9a44-5f4629f1ac4c)

![CleanShot 2024-03-26 at 02 38 38@2x](https://github.com/expo/expo/assets/10234615/424092ba-d572-48ca-9bcb-4f5bed60afaf)